lib/_http_outgoing.js

```diff
@@ -56,6 +57,7 @@ const HIGH_WATER_MARK = getDefaultHighWaterMark();
 const { CRLF, debug } = common;
 
 const kIsCorked = Symbol('isCorked');
+const kErrored = Symbol('errored');
```

Comment:
nit `kErrorEmitted` to more closely follow streams

Reply:
Should we initialize `kErrored` in the constructor? Or will that unnecessarily increate object size?

Reply:
I'm not adding that to the constructor exactly because of that. I can add it if we think it's worth adding, it's not a big deal anyway.

Reply:
Sounds good either way to me. Just checking if it was intentional.